pkg/controller/usersignup/usersignup_controller_test.go

```diff
@@ -207,6 +208,82 @@ func TestUserSignupFailedMissingNSTemplateTier(t *testing.T) {
 		})
 }
 
+func TestUserSignupFailedNoClusterReady(t *testing.T) {
+	// given
+	userSignup := &v1alpha1.UserSignup{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "foo",
+			Namespace: operatorNamespace,
+			UID:       types.UID(uuid.NewV4().String()),
+		},
+		Spec: v1alpha1.UserSignupSpec{
+			Username: "foo@redhat.com",
+			Approved: false,
+		},
+	}
+	r, req, _ := prepareReconcile(t, userSignup.Name, userSignup, configMap(config.UserApprovalPolicyAutomatic)) // basicNSTemplateTier does not exist
```

Comment:
```suggestion
	r, req, _ := prepareReconcile(t, userSignup.Name, userSignup, configMap(config.UserApprovalPolicyAutomatic))
```
I guess that the comment at the end of the line was result of copy-paste

Reply:
right, it's a valid comment but it is not relevant for this test, indeed. Removed in a0a3019